src/daikinAirConditioningAccessory.ts

```diff
@@ -41,32 +40,39 @@ export class DaikinCloudAirConditioningAccessory {
             .onGet(this.handleCurrentTemperatureGet.bind(this));
 
         this.service.getCharacteristic(this.platform.Characteristic.TargetHeaterCoolerState)
+            .setProps({
+                minStep: 1,
+                minValue: 0,
+                maxValue: 2,
+            })
             .onGet(this.handleTargetHeaterCoolerStateGet.bind(this))
             .onSet(this.handleTargetHeaterCoolerStateSet.bind(this));
 
         this.service.getCharacteristic(this.platform.Characteristic.CoolingThresholdTemperature)
             .setProps({
-                minStep: this.accessory.context.device.getData('climateControl', 'temperatureControl', '/operationModes/cooling/setpoints/roomTemperature').minStep,
-                minValue: this.accessory.context.device.getData('climateControl', 'temperatureControl', '/operationModes/cooling/setpoints/roomTemperature').minValue,
-                maxValue: this.accessory.context.device.getData('climateControl', 'temperatureControl', '/operationModes/cooling/setpoints/roomTemperature').maxValue,
+                minStep: accessory.context.device.getData('climateControl', 'temperatureControl', '/operationModes/cooling/setpoints/roomTemperature').minStep,
```

Comment:
Hello 👋 

This doesn't work if device is configured for control mode: "leavingWaterTemperature":

```json
"controlMode": {
            "settable": false,
            "requiresReboot": false,
            "value": "leavingWaterTemperature",
            "values": [
                "leavingWaterTemperature",
                "externalRoomTemperature",
                "roomTemperature"
            ]
        },
 ```
 
 --
 
 I think figuring out setpoints paths should be done dynamically based on control mode.
 
 Also for "leavingWaterTemperature" setpoint paths are different and set via "leavingWaterOffset":
 
 ```json
 "temperatureControl": {
            "/operationModes/auto/setpoints/leavingWaterOffset": {
                "settable": true,
                "requiresReboot": false,
                "value": 0,
                "maxValue": 10,
                "minValue": -10,
                "stepValue": 1
            },
            "/operationModes/heating/setpoints/leavingWaterOffset": {
                "settable": true,
                "requiresReboot": false,
                "value": 0,
                "maxValue": 10,
                "minValue": -10,
                "stepValue": 1
            },
            "/operationModes/cooling/setpoints/leavingWaterTemperature": {
                "settable": true,
                "requiresReboot": false,
                "value": 20,
                "maxValue": 22,
                "minValue": 5,
                "stepValue": 1
            }
        }
  ```
  
It also applies to sensory data readout:

```json
"sensoryData": {
            "/leavingWaterTemperature": {
                "settable": false,
                "requiresReboot": false,
                "value": 29,
                "maxValue": 127,
                "minValue": -127,
                "stepValue": 1
            }
        },
```